src/algebra/group_power/basic.lean

```diff
@@ -0,0 +1,145 @@
+/-
+Copyright (c) 2015 Jeremy Avigad. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Jeremy Avigad, Robert Y. Lewis
+-/
+import algebra.order_functions
+import deprecated.group
```

Comment:
```suggestion
```
Looks like this might not be needed until `lemmas.lean`? (Sorry, haven't actually compiled Lean 3.20 to test.)

Reply:
It isn't needed yet but I intend for it to be used in the second phase of this project: moving some `group_power` lemmas into this file so that they are available in `data.nat.basic`, allowing us to remove the redundant `nat.*` versions of those lemmas. Probably unnecessarily, I've added the import now in order to "reserve the place" in the module structure.